.ci/jenkins/Jenkinsfile

```diff
@@ -117,16 +128,31 @@ def Notify() {
     UTILS.CreateStage("Notify", notifyMethod, configurationMaps)
 }
 
-def main(String projectName, String projectRoot, String dockerContainerName) {
-    timeout(activity: true, time: 60) {
-        def configurationMaps = UTILS.GetDockerEnvList(projectName, dockerContainerName, projectRoot)
-        BuildImage(configurationMaps)
-        RunDockerContainers(configurationMaps)
-        PrepareEnvironment(configurationMaps)
-        RunToxTests(configurationMaps)
-        Cleanup(configurationMaps)
-        Notify()
+def main(String projectName, String projectRoot, String dockerContainerName, String jenkins_github_credential_id, String repository_git_address) {
+    node('ci && onnx') {
```

Comment:
maybe the node definition should be in  line 158 i.e. if we would like to change labels
```
node('ci && onnx') {
   main(PROJECT_NAME, CI_ROOT, DOCKER_CONTAINER_NAME, JENKINS_GITHUB_CREDENTIAL_ID, REPOSITORY_GIT_ADDRESS)
}
```
what do you think